src/usercmd.c

```diff
@@ -329,11 +329,7 @@ set_context_in_user_cmdarg(
 	return set_context_in_menu_cmd(xp, cmd, arg, forceit);
 #endif
     if (context == EXPAND_COMMANDS)
-    {
-	if (xp->xp_context == EXPAND_NOTHING)
-	    xp->xp_context = context;
```

Comment:
Is this change needed?  Will this break something else?

Reply:
It was just added two days ago.

Reply:
okay thanks